source/frontend/src/features/mapSideBar/disposition/tabs/fileDetails/detail/update/UpdateDispositionForm.tsx

```diff
@@ -244,10 +244,8 @@ const UpdateDispositionForm: React.FC<IUpdateDispositionFormProps> = ({
 export default UpdateDispositionForm;
 
 const Container = styled.div`
-  .form-section {
-    margin: 0;
-    padding-left: 0;
-  }
+  background-color: ${props => props.theme.css.filterBackgroundColor};
+  padding-top: 1rem;
```

Comment:
Fixing the styling of the Update Disposition form. Was missing the grey background and padding